### PR TITLE
Meenu/feat: created tabs and organised location

### PR DESCRIPTION
**Pull request by meenakshi-deriv**
Changes:
Organized locations under deriv/careers.
Filter countries based on continent.
-   ...

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/careers/_model/_locations/_locations.ts [hunk 1]
```diff
@@ -379,5 +390,23 @@ export const locations = {
     france: `${paris.display_name}, ${paris.country}`,
     guernsey: `${guernsey.display_name}, ${guernsey.country}`,
 }
+export const continents = {
+    All: all_offices,
```
Comment (prince-deriv) [inline src/pages/careers/_model/_locations/_locations.ts:394]:
```suggestion
    all: all_offices,
```

can we make this lowercase

Comment (meenakshi-deriv) [reply src/pages/careers/_model/_locations/_locations.ts]: Sure @prince-deriv  I have updated this one.

File: src/pages/careers/locations/index.tsx [hunk 2]
```diff
@@ -199,38 +204,63 @@ const Locations = () => {
...
-                            Icon={office.flagIcon}
-                            img_data={images[office.thumbnail]}
-                            country_name={office.country}
-                            city_name={office.display_name}
-                            link={office.link}
-                            img_alt={office.img_alt}
-                        />
-                    ))}
-                </CssGrid>
+                    {Object.keys(continents).map((key) => {
```
Comment (habib-deriv) [inline src/pages/careers/locations/index.tsx:222]: Please assign this value in a meaningful variable name `Object.keys(continents)` later you can just map over the variable.

File: src/pages/careers/locations/index.tsx [hunk 3]
```diff
@@ -199,38 +204,63 @@ const Locations = () => {
...
+                        return (
+                            <Tabs.Panel
+                                label={key
+                                    .split('_')
+                                    .filter((name) => name.length > 0)
+                                    .map((name) => name.charAt(0).toUpperCase() + name.slice(1))
```
Comment (habib-deriv) [inline src/pages/careers/locations/index.tsx:228]: Create a function for this calculation, so it's easy to understand what's happening.

File: src/pages/careers/locations/index.tsx [hunk 4]
```diff
@@ -188,6 +193,16 @@ const query = graphql`
 const Locations = () => {
     const images = useStaticQuery(query)
 
+    const continents = Object.keys(allContinents)
+
+    const formatContinentName = (continent) => {
+        return continent
+            .split('_')
+            .filter((continentName) => continentName.length > 0)
+            .map((continentName) => continentName.charAt(0).toUpperCase() + continentName.slice(1))
```
Comment (habib-deriv) [inline src/pages/careers/locations/index.tsx:202]: Just this Part, use a method for this
continentName.charAt(0).toUpperCase() + continentName.slice(1))

**Review by habib-deriv — APPROVED** [review]
LGTM

**Review by ShuvoHabib — APPROVED** [review]
LGTM

**Review by habib-deriv — DISMISSED** [review]
LGTM

**Pull request merged**